books/RayTracingTheRestOfYourLife.html

```diff
@@ -3595,9 +3595,9 @@
             return sum;
         }
 
-        vec3 random(const vec3& o) const override {
+        vec3 random(const vec3& origin) const override {
```

Comment:
The declaration in the base class (i.e. `hittable::random` and `hittable::pdf_value`) still use singleletter names. I think it makes sense to update the listing (Listing 38) and the code to match.

I also noticed that `sphere` (TheRestOfYourLife/sphere.h) also still uses the single-letter names, so I would update it to be consistent. AFAICT `sphere` is the only `hittable` subclass that needs to be updated but I recommend double checking all of them in case I missed one.

Reply:
Will do.